### PR TITLE
feat(replays): Add accessibility-issues blueprint

### PR DESCRIPTION
**Pull request by cmanallen**

### CODE REVIEW

File: src/sentry/replays/blueprints/api.md [hunk 1]
```diff
@@ -238,6 +238,47 @@ Deletes a replay instance.
...
+- Response 200
+
+  ```json
+  {
+    "meta": {
+      "total": 1
+    },
+    "data": [
+      {
+        "id": "AHC203525",
```
Comment (ryan953) [inline src/sentry/replays/blueprints/api.md:270]: is this an issue id, specific to the detected issue type, mapping to the description/help_url/summary?

Comment (cmanallen) [reply src/sentry/replays/blueprints/api.md:270]: Unclear to me at the moment.  I just copied what was on the hackweek project.  Its a todo for me to get in there and figure out exactly what axe-core is returning.  the other fields are pretty self-explanatory i feel but `id` is definitely not.

Comment (ryan953) [reply src/sentry/replays/blueprints/api.md:270]: np np

File: src/sentry/replays/blueprints/api.md [hunk 2]
```diff
@@ -238,6 +238,91 @@ Deletes a replay instance.
...
+| id      | string | String ID of the accessibility issue. |
+| message | string | Message explaining the problem.       |
+
+- Response 200
+
+  ```json
+  {
+    "meta": {
+      "total": 1
+    },
```
Comment (ryan953) [inline src/sentry/replays/blueprints/api.md:283]: lets flip this over to the `X-Hits` header because that seems to be the built-in way

**Review by ryan953 — APPROVED** [review]

**Pull request merged**